### PR TITLE
Always use ticker symbol

### PR DESCRIPTION
**Pull request by egh**
- Move CUSIP to ticker mapping to OfxConverter
- Always use ticker symbol if available

### CODE REVIEW

**Comment by egh** [comment]
@jblachly Does this change make sense to you?

File: ledgerautosync/converter.py [hunk 1]
```diff
@@ -192,6 +192,13 @@ def __init__(self, ofx, name, indent=4, ledger=None, fid=None,
                                            unknownaccount=unknownaccount,
                                            currency=ofx.account.statement.currency)
         self.acctid = ofx.account.account_id
+        # build SecurityList (including indexing by CUSIP and ticker symbol)
+        if hasattr(ofx, 'security_list') and ofx.security_list is not None:
+            self.security_list = SecurityList(ofx.security_list)
+        else:
+            self.security_list = SecurityList([])
+
+            self.acctid = ofx.account.account_id
```
Comment (jblachly) [inline ledgerautosync/converter.py]: I think this line is a duplicate of line 194 ?

**Comment by jblachly** [comment]
@egh, interestingly that is the original solution I created (in my comments I referred to "conversion at point-of-use"). This does have the advantage of not relying on "magic," but does mean more potential work in the future. For example, the commit in this pull request does not change the ticker symbol in the position statement printouts:

https://github.com/egh/ledger-autosync/blob/5a856f71352a44db38b3108ea97895d899e67d18/ledgerautosync/converter.py#L379

So, you'll need to do the same cusip->ticker remapping inside that function ... but also anywhere else that in the future uses the ticker symbol. If you'e okay with that, then roll with it because I admit the remapping code that rewrites the OFX object is ugly!

File: ledgerautosync/converter.py (hunk 1 above)
Comment (egh) [inline ledgerautosync/converter.py]: Oops! Fixed.

**Comment by egh** [comment]
@jblachly Great, thanks! I thought that might have been what you'd originally coded. I think it's a little simpler and it shouldn't be slower. Adapting it to price statements was pretty easy as well.

**Comment by jblachly** [comment]
@egh looks great, love the function name!  LGMT, Ship it!

**Comment by egh** [comment]
Great, thanks for taking the time to look at over and find what I missed! I really appreciate all the help.

**Pull request merged**